policyengine_us/variables/gov/ssa/ss/never_eligible_for_social_security_benefits.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class never_eligible_for_social_security_benefits(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = "Never eligible for Social Security"
+    unit = USD
+
+    def formula(person, period, parameters):
+        # Assumption: if person is above age 70 and has no social security benefits, then they are never eligible.
```

Comment:
@MaxGhenis Is this what you meant by encoding an assumption separately from a rule? Looks like even though the full retirement age has changed over time, 70 as the latest age to delay benefits to has not.

Reply:
Yep thanks